package.json

```diff
@@ -23,5 +23,8 @@
     "verbose": true,
     "testURL": "http://localhost/",
     "testEnvironment": "node"
+  },
+  "dependencies": {
```

Comment:
Your PR text said adding the package to webapp but it's installed here in root. Either functions, but I think webapp as your note said makes more sense.